hotness/tests/test_consumer.py

```diff
@@ -0,0 +1,130 @@
+"""
+Unit tests for hotness.consumer
+"""
+from __future__ import unicode_literals, absolute_import
+
+import unittest
+
+import mock
+
+import fedmsg.config
+
+from hotness import consumers
+
+
+class TestConsumer(unittest.TestCase):
+    def setUp(self):
+        self.bz = mock.patch('hotness.bz.Bugzilla')
+        self.bz.__enter__()
+        self.koji = mock.patch('hotness.buildsys.Koji')
+        self.koji.__enter__()
+
+        test_config = fedmsg.config.load_config()
+        test_config["hotness.cache"] = {
+            "backend": "dogpile.cache.null",
+        }
+        test_config['hotness.requests_retries'] = 0
+
+        class MockHub(mock.MagicMock):
+            config = test_config
+
+        self.consumer = consumers.BugzillaTicketFiler(MockHub())
+        import logging
+        logging.basicConfig(level=logging.DEBUG)
+
+    def tearDown(self):
+        self.bz.__exit__()
+        self.koji.__exit__()
+
+    def test_is_retired_negative(self):
+        """ python-sqlite2 was rolled into the stdlib.  retired. """
+        response = mock.MagicMock()
+        response.status_code = 200
+        response.json.return_value = {'count': 0}
+        self.consumer.requests_session = mock.MagicMock()
+        self.consumer.requests_session.get.return_value = response
+
+        package = 'python-sqlite2'
+        expected = True
+        actual = self.consumer.is_retired(package)
+        self.assertEquals(expected, actual)
+
+    def test_is_retired_positive(self):
```

Comment:
I find the test names a little confusing when combined with ``is_retired`` (which could probably just be called ``retired``). To me positive implies "Yes, it _is_ retired". I don't mind one way or the other since the docblock clears it up, but just so you know.

Reply:
Yeah - this is confusing.  Will address.

Reply:
Adjusted in a5bfa03.